pkg/domain/infra/abi/play.go

```diff
@@ -72,7 +76,8 @@ func (ic *ContainerEngine) createServiceContainer(ctx context.Context, name stri
 		ReadOnly:         true,
 		ReadWriteTmpFS:   false,
 		// No need to spin up slirp etc.
-		Net: &entities.NetOptions{Network: specgen.Namespace{NSMode: specgen.NoNetwork}},
+		Net:         &entities.NetOptions{Network: specgen.Namespace{NSMode: specgen.NoNetwork}},
+		StopTimeout: rtc.Engine.StopTimeout,
```

Comment:
Nice catch!

Reply:
Reporter actually caught it.

Reply:
@ InvalidInterrupt  PTAL

Reply:
Thinking about it without a fried brain:  I am not sure the stop timeout makes that much sense for a service container.  There is no workload inside other than catatonit.  But catatonit handles SIGTERM just fine so I don't expect any measurable negative performance impact on shut down.